app/src/common/shared/org/mozilla/vrbrowser/browser/SettingsStore.java

```diff
@@ -305,4 +306,15 @@ public void setMSAALevel(int level) {
         editor.commit();
     }
 
+    public String getVoiceSearchLanguage() {
+        return mPrefs.getString(
+                mContext.getString(R.string.settings_key_voice_search_language), LocaleUtils.getCurrentLocale());
```

Comment:
I don't see `voice_search_language` in `strings.xml` in here or in our [`strings.xml`](https://github.com/MozillaReality/FirefoxReality/blob/096424app/src/main/res/values/strings.xml
) in our PR #711 (but I do see it in the [Wiki page](https://github.com/MozillaReality/FirefoxReality/wiki/L10n#voice_search_language) - thanks!)

Reply:
As per our conversation in Slack we will need to keep those language related strings here for the moment.

Regarding **settings_key_voice_search_language** that's a settings key so not being a translatable string it lives in **non_L10n.xml**.

Reply:
@cvan can you also provide a longer logcat, I don't seem to be able to reproduce that crash in local. I think the real crash is a bit above the one that you sent.

Reply:
happened again on another Oculus Go headset:

https://gist.github.com/cvan/675cff1162dc9f5a4cf8153ab3627be8

not sure what to sift through from that

Reply:
@cvan The crash should be fixed.